OpenUpSA/wazimap-ng-ui
# Updated tutorial popup to be dynamic

## Description

Make tutorial popup dynamic

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-859

## How is it tested automatically?

## How should a reviewer test it locally
* By default tutorial with 8 pages should show up
* But if in admin we add a page with selected profile and Api mapping as `tutorial`, We will be able to override the tutorial popup data
* to add pages in pop up add contents for page

Admin panel link :
Page : https://staging.wazimap-ng.openup.org.za/admin/cms/page/
content : https://staging.wazimap-ng.openup.org.za/admin/cms/content/

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
